audio-experience/src/main/kotlin/tw/waterballsa/utopia/audiox/MuteAudiences.kt

```diff
@@ -0,0 +1,66 @@
+package tw.waterballsa.utopia.audiox
+
+import net.dv8tion.jda.api.entities.Widget.VoiceChannel
+import net.dv8tion.jda.api.events.interaction.command.SlashCommandInteractionEvent
+import net.dv8tion.jda.api.interactions.commands.OptionType
+import net.dv8tion.jda.api.interactions.commands.build.Commands
+import net.dv8tion.jda.api.interactions.commands.build.SubcommandData
+import tw.waterballsa.utopia.commons.config.WsaDiscordProperties
+import tw.waterballsa.utopia.jda.listener
+
+private const val OPTION_AUDIENCE_NAME = "audience"
+private const val OPTION_ROLE_NAME = "role"
+
+fun muteAudiences(wsa: WsaDiscordProperties) = listener {
+    // hello world <arg1>
+    // <feature-module's name> <feature's name> <arg1> <arg2> ....
+
+    // mute audiences
+    // allow alpha members
+    // allow speakers
```

Comment:
give a more comprehensive doc about how to use this command

Reply:
/*
    

- mute audiences: The command allows the user to mute audiences
Optional -> audience: Allow the audience to unmute
Optional -> role: Allow all members of a role to unmute

- mute revoked: The command allows the user to revoke mute

 */